frontstage/controllers/case_controller.py

```diff
@@ -197,19 +145,19 @@ def get_cases_by_party_id(party_id, case_events=False):
     return response.json()
 
 
-def get_eq_url(case_id, party_id):
+def get_eq_url(case_id, party_id, business_party_id, survey_short_name):
     logger.debug('Attempting to generate EQ URL', case_id=case_id, party_id=party_id)
 
     case = get_case_by_case_id(case_id)
 
-    if case['caseGroup']['caseGroupStatus'] == 'COMPLETE':
+    if case['caseGroup']['caseGroupStatus'] == 'COMPLETE' or case['caseGroup']['caseGroupStatus'] == 'COMPLETEDBYPHONE':
         logger.info('The case group status is complete, opening an EQ is forbidden',
                     case_id=case_id, party_id=party_id)
         abort(403)
 
-    check_case_permissions(party_id, case['partyId'], case_id=case_id)
+    check_case_permissions(party_id, case['partyId'], business_party_id, survey_short_name)
 
-    payload = eq_payload.EqPayload().create_payload(case)
+    payload = eq_payload.EqPayload().create_payload(case, party_id, business_party_id, survey_short_name)
```

Comment:
Should we just import the `EqPayload` class rather than the whole `eq_payload` file?